<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignupOrganization</class>
 <widget class="QMainWindow" name="SignupOrganization">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>968</width>
    <height>737</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_back">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>back</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>660</width>
      <height>660</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox
{
border-radius: 10px;
background-color: rgb(43, 43, 43);
}

QLineEdit
{
background-color: rgb(81, 81, 81);
border: none;
border-radius: 5px;
border-bottom: 2px solid #9c9c9c;
color: rgb(214, 214, 214);
}

QPushButton
{
background-color: rgb(68, 68, 68);
color: rgb(214, 214, 214);
border-radius: 5px;
}

QDateEdit
{
background-color: rgb(81, 81, 81);
border: none;
border-radius: 5px;
border-bottom: 2px solid #9c9c9c;
color: rgb(214, 214, 214);
}

QTextEdit
{
background-color: rgb(81, 81, 81);
border: none;
border-radius: 5px;
border-bottom: 2px solid #9c9c9c;
color: rgb(214, 214, 214);
}
</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>305</x>
       <y>50</y>
       <width>71</width>
       <height>70</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../icons/icons8-signup-64(7).png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>275</x>
       <y>130</y>
       <width>110</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 66, 198);</string>
     </property>
     <property name="text">
      <string>signup</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ln_username">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>220</y>
       <width>160</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="placeholderText">
      <string> username</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ln_country">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>280</y>
       <width>160</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="placeholderText">
      <string> country</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ln_phonenumber">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>340</y>
       <width>160</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="placeholderText">
      <string> phonenumber</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ln_password">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>280</y>
       <width>160</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string> password</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ln_name">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>220</y>
       <width>160</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="placeholderText">
      <string> name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>220</y>
       <width>160</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="placeholderText">
      <string> organization ceo </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_header">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>340</y>
       <width>160</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>header</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ln_link">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>280</y>
       <width>160</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string> link</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="txt_bio">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>340</y>
       <width>160</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:7.8pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="placeholderText">
      <string>biography</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_signup">
     <property name="geometry">
      <rect>
       <x>285</x>
       <y>520</y>
       <width>90</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Dubai</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 66, 198);</string>
     </property>
     <property name="text">
      <string>done</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../icons/icons8-done-50.png</normaloff>../../icons/icons8-done-50.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>400</y>
       <width>160</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Dubai</family>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 66, 198);</string>
     </property>
     <property name="text">
      <string>import profile image</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>968</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
